buildSrc/src/main/kotlin/ivy.lint.gradle.kts

```diff
@@ -0,0 +1,47 @@
+import io.gitlab.arturbosch.detekt.Detekt
+import io.gitlab.arturbosch.detekt.extensions.DetektExtension
+
+plugins {
+    id("io.gitlab.arturbosch.detekt")
+}
+
+configure<DetektExtension> {
+    source.setFrom(projectDir)
+    config.setFrom("$rootDir/config/detekt/config.yml")
+    baseline = file("$rootDir/config/detekt/baseline.yml")
+}
+
+tasks.register<Detekt>("detektFormat") {
+    autoCorrect = true
+}
+
+tasks.withType<Detekt> {
+    // Disable task caching
+    outputs.upToDateWhen { false }
+
+    reports {
+        html.required.set(true)
+        sarif.required.set(false)
+        md.required.set(false)
+        xml.required.set(false)
+        txt.required.set(false)
+    }
+
+    include("**/*.kt", "**/*.kts")
+    exclude("**/resources/**", "**/build/**")
+
+    parallel = true
+
+    buildUponDefaultConfig = true
+
+    allRules = true
+
+    // Target version of the generated JVM bytecode. It is used for type resolution.
+    jvmTarget = "1.8"
```

Comment:
Can, but what the reason? 

It's classical gradle code. You will likely never change jvm target to 11.

Reply:
We may, in some other projects that I've participated in, we even used Java 18.
This might be to be extracted in `:app` and `:ivy-design`. It's not blocking though - I can do it later

Reply:
It's not about jvm version, it's about target bytecode. 

How do you use jvm18 bytecode with android? 

Android 13 has access to Java 8, 9, 11, and 17
Android 11 and 12 has access to Java 8, 9, and 11
Android Pie (9) and 10 has access to Java 8 and 9

Our minSdk is 28 -> android 9

Reply:
17* my bad

Reply:
Let's merge it - it's a big improvement we won't bump.Java anytime soon prolly

Reply:
So yes we can bump to java 9, but... And yes you can extract it to one place. Really change will be when min supported android version will be 11.